Automattic/jetpack
# Untangle: Move Subscribers to Jetpack

Fixes https://github.com/Automattic/dotcom-forge/issues/5445

## Proposed changes:
Move Subscribers from Users menu to Jetpack menu, and add an external link icon.

| Before | After |
| --- | --- |
| <img width="1136" alt="image" src="https://github.com/Automattic/jetpack/assets/402286/4022e5ee-7c40-4652-845c-05015dd53b98"> | <img width="1114" alt="image" src="https://github.com/Automattic/jetpack/assets/402286/d9ef1ebf-2878-49e9-9ab7-e4b2e9183650"> |

## Jetpack product discussion
Details on Issue: https://github.com/Automattic/dotcom-forge/issues/5445

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:
* Set your `Admin interface style` to `Classic (wp-admin)` in `wordpress.com/hosting-config/site_slug`
* Go to `wp-admin`
* The Jetpack menu should have the Subscribers item with an external link icon.
* The Users menu should not have the Subscribers item.
